agents/imap/general.py

```diff
@@ -3,78 +3,85 @@
-		
-	def Load(self):
-		if not os.path.isfile(self.FileName):
-			self.Init()
-		else:
-			self.Set(File().GetAsList(self.FileName)[0].strip())
-	
-	def Save(self):
-		#print "Saving Storage " + str(self.Content) + ".."
-		File().Save(self.FileName, self.Content)
+    def __init__(self, FileName):
+        self.FileName = FileName
+        self.Init()
+        self.Load()
+
+    def Init(self):
+        self.Content = ""
+
+    def Get(self):
+        return self.Content
+
+    def Set(self, Content):
+        self.Content = Content
+
+    def Load(self):
+        if not os.path.isfile(self.FileName):
+            self.Init()
+        else:
+            self.Set(File().GetAsList(self.FileName)[0].strip())
+
+    def Save(self):
+        File().Save(self.FileName, self.Content)
+
 
 class File:
-	def GetAsList(self, FileName):
-		try:
-			Output = open(FileName, 'r').read().split("\n")
-			print "Loaded file: '"+FileName+"'"
-		except IOError, error:
-			print "Cannot open file: '"+FileName+"' ("+str(sys.exc_info())+")"
-			Output = []
-		return Output
-	
-	def Save(self, FileName, Data):
-		Data = str(Data)
-		try:
-			print "FileName=" + str(FileName) + ", Data=" + str(Data)
-			print "Saving to File '" + str(FileName) + "' " + str(Data) + ".."
-			File = open(FileName, 'w')
-			File.write(Data)
-		except IOError, error:
-			print "Cannot write to: '" + FileName +"' ("+str(sys.exc_info())+")"
+    def GetAsList(self, FileName):
+        try:
+            Output = open(FileName, 'r').read().split("\n")
+            print "Loaded file: '%s'" % FileName
+        except IOError:
+            print "Cannot open file: '%s' (%s)" % (FileName, str(sys.exc_info()))
+            Output = []
+        return Output
+
+    def Save(self, FileName, Data):
+        Data = str(Data)
+        try:
+            print "FileName=%s, Data=%s" % (FileName, str(Data))
+            print "Saving to File '%s' %s.." % (FileName, str(Data))
+            File = open(FileName, 'w')
+            File.write(Data)
+        except IOError:
+            print "Cannot write to: '%s' (%s)" % (FileName, str(sys.exc_info()))
+
 
 class Config:
-	def __init__(self, FileName):
-		self.Config = defaultdict(list)
-		self.Load(FileName)
-		
-	def Load(self, FileName):
-		for Line in File().GetAsList(FileName):
-			try:
-				Name = Line.split(":")[0]
-				Value = Line.replace(Name + ":", "").strip()
-				self.Config[Name] = Value
-			except:
-				print "Cannot parse line: " + Line
-				
-	def Get(self, Setting):
-		return self.Config[Setting]
-	
-	def Set(self, Setting, Value):
-		self.Config[Setting] = Value
+    def __init__(self, FileName):
+        self.Config = defaultdict(list)
+        self.Load(FileName)
+
+    def Load(self, FileName):
+        for Line in File().GetAsList(FileName):
+            try:
+                Name = Line.split(":")[0]
+                Value = Line.replace("%s:" % Name, "").strip()
+                self.Config[Name] = Value
+            except:
```

Comment:
Catching all exceptions is really a bad habit. Could you update the `except` to catch only the required exceptions?